nikola/plugins/shortcode/listing.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+# This file is public domain according to its author, Brian Hsu
```

Comment:
Whoa, flashback. I don't think any of that code is still there. OTOH, I suspect this is because I copied gist.py :-)